### PR TITLE
Fix bug in m2rotaxis

### PR DESCRIPTION
**Pull request by RasmusFonseca**
Takes 180deg rotation into account and added some documentation and tests.

### CODE REVIEW

File: Bio/PDB/Vector.py [hunk 1]
```diff
@@ -285,11 +305,20 @@ def normsq(self):
         return abs(sum(self._ar * self._ar))
 
     def normalize(self):
-        """Normalize the Vector."""
+        """
+        Normalize the Vector object.
```
Comment (peterjc) [inline Bio/PDB/Vector.py]: Please put the one-line docstring summary on the same line as the opening triple quote, as per examples in PEP257 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

**Comment by RasmusFonseca** [comment]
@peterjc Yes, you can merge his first, but there will be a minor conflict that needs to be resolved in the beginning of `rotmat`. Nothing big. 

Thanks for pointers on style guide. Ill go through it at some point today.

**Comment by peterjc** [comment]
OK, I manually applied #494 - the merge conflict was due to a minor docstring change.

Do you want to rebase this branch against the new master, or should I do this?

**Comment by RasmusFonseca** [comment]
It'd be great if you could. I haven't rebased before, and looking it up indicates there are enough steps that I could mess something up ;) Travis fails because I should have run `flake8 Tests` as well. Ill fix those first.

**Comment by peterjc** [comment]
Sorry - this is a new style issue, TravisCI checks for this were enabled only earlier today:

```
$ flake8 Bio/
Bio/PDB/Vector.py:24:9: E129 visually indented line with same indent as next logical line
Bio/PDB/Vector.py:29:5: E129 visually indented line with same indent as next logical line
```

See 73896e47ec77bc158990a24f777a44b2c8490d43 - long lines might be an acceptable solution if you can't see how to break the long line nicely.

**Comment by RasmusFonseca** [comment]
Haha. The wonders of trying to commit to an active project. I made them single line.

**Comment by peterjc** [comment]
I've manually applied this to the master branch, and included you in the acknowledgements. Thank you.

Apologies if I already asked, but are you happy to dual licence your contributions under both the _Biopython License Agreement_ AND the _BSD 3-Clause License_? See #898.

**Comment by RasmusFonseca** [comment]
You didn't ask, but I'm happy to appear as a contributor and I agree to the licenses.

**Comment by peterjc** [comment]
Thank you

**Pull request closed without merging**